project/dexter-monitor/public/ctrl/currentProjectCtrl.js

```diff
@@ -0,0 +1,116 @@
+            headerTooltip: 'Number of fixed defects', aggregationType: uiGridConstants.aggregationTypes.sum},
+        {field:'defectCountDismissed',  displayName:'Dismissed',        width: '9%',    cellClass: 'grid-align',
+            headerTooltip: 'Number of dismissed defects', aggregationType: uiGridConstants.aggregationTypes.sum},
+        {field:'resolvedRatio',         displayName:'Resolved ratio',   width: '11%',   cellClass: 'grid-align',
+            headerTooltip: '(Fixed + Dismissed) / Total * 100', footerCellClass: 'grid-align',
+            aggregationType: () => `${$scope.resolvedRatioTotal}`,
+            cellTemplate:'<div class="ui-grid-cell-contents">{{grid.appScope.getResolvedRatio(row.entity)}}</div>'},
+        {field:'serverStatus',          displayName:'Server Status',    width: '16%',   cellClass: 'grid-align',
+            headerTooltip: 'Server status', footerCellClass: 'grid-align',
+            aggregationType: () => `Active: ${$scope.activeServerCount} / Inactive: ${$scope.allServerCount-$scope.activeServerCount}`,
+            cellTemplate:'<div class="ui-grid-cell-contents"' +
+                        ' ng-class="{\'server-status-active\':COL_FIELD == \'Active\',' +
+                        '            \'server-status-inactive\':COL_FIELD == \'Inactive\'}">{{COL_FIELD}}</div>'}
+    ];
+
+    initialize();
+
+    function initialize() {
+        $scope.resolvedRatioTotal = '';
+        $scope.allServerCount = 0;
+        $scope.activeServerCount = 0;
+        $scope.gridOptions = createGrid(columnDefs);
+        $scope.gridOptions.showColumnFooter = true;
+        refreshData();
+        $scope.refreshDataTimer = $interval(refreshData, 10000);
+    }
+
+    function refreshData() {
+        if ($scope.dotAdder) {
+            $interval.cancel($scope.dotAdder);
+        }
+        ServerStatusService.getActiveServerList()
+            .then(loadData)
+            .catch((err) => {
+                $log.error(err);
+            });
+        $scope.time = new Date().toLocaleString();
+        $scope.dots = '.';
+        $scope.dotAdder = $interval(() => {$scope.dots += '.';}, 1000);
+        setGridExportingFileNames($scope.gridOptions, CURRENT_STATUS_FILENAME_PREFIX + '-' + $scope.time);
+    }
+
+    function loadData(activeServerList) {
+        ProjectService.getAllCurrentStatusList(activeServerList)
+            .then((rows) => {
+                $scope.gridOptions.data = rows;
+                const defectCountTotalSum = _.sum(_.map($scope.gridOptions.data, 'defectCountTotal'));
+                const defectCountFixedSum = _.sum(_.map($scope.gridOptions.data, 'defectCountFixed'));
+                const defectCountDismissedSum = _.sum(_.map($scope.gridOptions.data, 'defectCountDismissed'));
+                if (defectCountTotalSum > 0) {
+                    $scope.resolvedRatioTotal = `${((defectCountFixedSum + defectCountDismissedSum) / defectCountTotalSum * 100).toFixed(1)}%`;
```

Comment:
Please separated by a function like as getResolvedRatio().
When we have to change calculation for ratio, We have to check only this function.